### DISCUSSION TITLE
Included in Ubuntu 22.04 LTS

### DISCUSSION THREAD

**Discussion by eeickmeyer** [phisch/phinger-cursors#9] [Show and tell]
Hi there!

My upload of phinger-cursors to Ubuntu was approved. It'll be in the Universe repository and part of Ubuntu Studio by default. The package name is phinger-cursor-theme to fit with the naming convention of other cursor themes in the repository.

That to say, congratulations!

**Comment by phisch** [comment]
That is amazing! I hope people will enjoy it, and thank you for packaging!

**Comment by phisch** [comment]
Hey @eeickmeyer I released [v2.0](https://github.com/phisch/phinger-cursors/releases/tag/v2.0). Do you mind updating the package whenever you have some time?

**Reply by wagrod** [reply]
I'm also waiting for the update.
